commands/blue_green_deploy_command.go

```diff
@@ -8,7 +8,10 @@ import (
 	"github.com/cloudfoundry/cli/plugin"
 )
 
-const noConfirmOpt = "no-confirm"
+const (
+	noConfirmOpt = "no-confirm"
+	keepExistingNamesOpt = "keep-existing-names"
```

Comment:
From the name of the option there is no clear way of understanding which names would be kept(names of applications or services). Maybe the option should be extended in order to contain more information about which names will be kept.